src/draco/io/stl_decoder.cc

```diff
@@ -62,21 +62,32 @@ Status StlDecoder::ParseHeader(bool* is_binary) {
   if (! buffer_.Decode(ascii_buffer, 5)) {
     return Status(Status::IO_ERROR, "STL file has invalid header.");
   }
+  bool is_ascii_file = ! strncmp(ascii_buffer, "solid ", 5);
```

Comment:
I think I have seen ascii stl files in the wild that start with "solid\n" or "solid\r".